core/injector/syncing_node_injector.hpp

```diff
@@ -15,7 +15,8 @@
 namespace kagome::injector {
   namespace di = boost::di;
 
-  auto get_peer_info = [](const auto &injector,
+  template<typename Injector>
+  auto get_peer_info(const Injector &injector,
```

Comment:
```suggestion
  auto getPeerInfo(const Injector &injector,
```

Reply:
It's not mine function, I just fixed it to be compiled with ```gcc10```.